extras/devices/LVFS/tests/test_LVFS.py

```diff
@@ -0,0 +1,66 @@
+import unittest
+import json
+from devices import *
+from unittest.mock import patch
+from devices.LVFS.tests import fwupd_data
+
+device_results = json.loads(fwupd_data.get_results)
+
+
+class TestLVFSDevice(unittest.TestCase):
+    def mock_run_cmd(*args, **kwargs):
+        if "sudo fwupdmgr get-results" in args[1]:
+            return 0, json.dumps(device_results), ""
+        return 0, "", ""
+
+    # current System Firmware version is the previous version
+    def test_upgradable(self):
+        with patch("devices.LVFS.LVFS.LVFSDevice.run_cmd") as mock_path:
+            mock_path.side_effect = self.mock_run_cmd
+            device = LVFSDevice("", "", "")
+            print(type(fwupd_data.get_device))
```

Comment:
Generally, using print() in test cases is not very helpful. I'm guessing you were trying to get some debug info with this? Did you intend to leave it?

Reply:
Oh I forgot to remove it.